packages/core/admin/ee/server/controllers/workflows/stages/index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const { getService } = require('../../../utils');
+
+module.exports = {
+  /**
+   * List all stages
+   * @param {import('koa').BaseContext} ctx - koa context
+   */
+  async find(ctx) {
+    const { id } = ctx.params;
```

Comment:
I think this should be workflowId instead
```suggestion
    const { workflow_id } = ctx.params;
```